modules/dashboard/test/TestPlan.md

```diff
@@ -48,32 +48,35 @@
 14. Verify that if a user has 'Across all sites access candidate profiles' permission, the reports works and even
     more important to check without this permission.
     [Automate Test]
-15. Verify that if a user has 'User Management / Survey Participant Management' permission, the number of pending
+15. Verify that if a user has 'issue_tracker_reporter / issue_tracker_developer' permission,
+    the issue tracker panel should be found without this permission.
+    [Automate Test on Travis_CI]
```

Comment:
@kongtiaowang 

this is confusing :)
Should the usre have either one or both? and did you mean to say the issue tracker panel should NOT be found?